ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -287,7 +287,10 @@ public void setViewState(@NonNull T view, @Nullable ReadableMap accessibilitySta
       view.setSelected(false);
     }
     view.setTag(R.id.accessibility_state, accessibilityState);
-    view.setEnabled(true);
+    
```

Comment:
There is some leading whitespace that is triggering an internal linter. Could you remove it?